scripts/remove-tests.sh

```diff
@@ -8,12 +8,26 @@ test "$?" -eq "0" && { echo "This script should only be executed." >&2; exit 1;
 set -Eeuo pipefail
 
 declare mydir
-mydir=$(pwd -P)
+mydir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
 
-echo ${mydir}
+declare HOPR_LOG_ID="remove tests"
+source "${mydir}/utils.sh"
 
-# Remove built tests and source maps
-rm -Rf ${mydir}/lib/**/*.spec.js ${mydir}/lib/*.spec.js
-rm -Rf ${mydir}/lib/**/*.spec.js.map ${mydir}/lib/*.spec.js.map
-rm -Rf ${mydir}/lib/**/*.spec.d.ts ${mydir}/lib/*.spec.d.ts
-rm -Rf ${mydir}/lib/**/*.spec.d.ts.map ${mydir}/lib/*.spec.d.ts.map
+usage() {
+  msg
+  msg "Usage: $0 [<pkg_dir>]"
+  msg
+  msg "\t<pkg_dir> package directory, e.g. 'packages/utils'"
+  msg
+}
+
+([ -z ${1:-} ] || [ "${1:-}" = "-h" ] || [ "${1:-}" = "--help" ]) && { usage; exit 0; }
+
+declare packagedir
+packagedir=$1
+
+# Remove built unit tests and source maps from given directory
+rm -Rf ${mydir}/../${packagedir}/lib/**/*.spec.js ${mydir}/../${packagedir}/lib/*.spec.js
```

Comment:
I'd rather use `find ${mydir}/../${packagedir}/lib -name "*.spec.js" -delete` for better failure reports and composability.

Reply:
changed